project1/index.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Document</title>
+  <link rel="stylesheet" href="styles.css">
+  <link href="https://fonts.googleapis.com/css?family=Lato:400,400i,700|Playfair+Display:400,900&display=swap"
+    rel="stylesheet">
+</head>
+
+<body>
+  <main>
+    <h1>The Crane Wife</h1>
+
+    <p class="authorinfo"><i> <a href="https://cjhauser.com/">By CJ Hauser</a> </i> - July 16, 2019</p>
```

Comment:
Per the [MDN documentation](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/i), the `i` element:

> represents a range of text that is set off from the normal text for some reason. Some examples include technical terms, foreign language phrases, or fictional character thoughts.

Given that, I don't think it's appropriate here and would recommend a `span` that you add CSS for to make this text italic.